README.md

```diff
@@ -51,6 +51,15 @@ SslCertificate::download()
    ->forHost($hostName);
 ```
 
+You can also check the certificate on a different IP address using the same style.
+```php
+SslCertificate::download()
+   ->setIpAddress($ipAddress)
```

Comment:
Let's rename the function to `fromIpAddress`, feels a bit more human.

Reply:
Agreed, done.